### ISSUE TITLE
Additional weather data

### ISSUE THREAD

**Issue by PGorzalka** [RWTH-EBC/AixLib#1558]
In a side discussion related to #1080 , @FWuellhorst @DaJansenGit and me were thinking about providing the soil temperature using the weather file. However, the current weather data reader from IBSPA does not support soil temperature. Similarly, it does not cover the longwelled radiation terms provided by the DWD TRY files.

I'd like to open this issue to have a discussion about if these data are needed and if some extension to the weather data reader is a good idea.

Benefits already discussed:
- no building-individual extra files needed for soil temperature

Potential drawbacks:
- soil temperature may actually be intended to be modelled individually for each building due to basement temperatures
- more complicated weather data handling

**Comment by FWuellhorst** [comment]
@PGorzalka I will check with IBPSA and see how they handle it. However, we could always extend TMY reader and add the soil temperature to the bus, if we agree on a format. I would like the concept, also for ground-source heat pumps o.s.

**Comment by PGorzalka** [comment]
@FWuellhorst is this something still to be worked on? 
- I have code in use internally that supports DWD TRY's longwelled radiation (from sky and ground) which could potentially be integrated into AixLib and maybe also to AixWeather if you'd like.
- for soil temperature, I suppose it's not suitable for AixWeather and I still don't know if it's suitable here. What is your current approach for ground-source heat pumps?

**Comment by FWuellhorst** [comment]
I think the soil temperature is to much of a guess to include by default and data is missing to include it in AixWeather. I don't focus on ground-source heat pumps, so I did not have the issue yet.
Regarding the other radiation data: What models are they useful for?

**Comment by PGorzalka** [comment]
For me soil temperature is also not an issue, so that's fine.

The other radiation data are so far only used partly. Sky radiation is considered in the calculation of the black sky temperature. Ground radiation is not, but for vertical building parts (i. e. walls) it makes up half of the radiation temperature in exchange with the surface. So including it would allow to improve the modelling of longwelled radiation exchange.

**Comment by FWuellhorst** [comment]
mhm ok, but that would require a change in the ROM model, or?

**Comment by PGorzalka** [comment]
Indeed, it requires changes to the IBPSA equivalent air temperature calculation (`AixLib.ThermalZones.ReducedOrder.EquivalentAirTemperature.VDI6007`)
I brought this up [here](https://github.com/ibpsa/modelica-ibpsa/issues/1744#issuecomment-1638415658) a while ago, but didn't follow up on it. Michael's argument was that the data for it is not available, so my thought was that the data needs to be made available first (here) ;)

**Comment by FWuellhorst** [comment]
In that case I would go for AixWeater -> IBPSA first, because if its not in the TMYReader I guess people will not use it.
